docs/alincome/expenses.md

```diff
@@ -0,0 +1,271 @@
+sidebar_label: |
+  Expenses
+slug: /alincome/expenses
+---
+
+The `ALExpenseList` is a class that lets you ask a user about their different expenses in a simple way.
+
+## Guiding Philosophy
+
+Certain court forms ask for a person's expenses as a proof of hardship or burden.
+In designing this feature, we tried to reduce the number of questions a user needs
+to answer while still asking for specific enough expenses. We don't want to burden the user, but also
+doesn't want the user forget to list any significant expenses they would have
+that could prove financial burden.
+
+What expenses you want to initially ask about will depend on your community's needs.
+For example, if your community lives in northern rural areas, they might have alternative
+heating needs, like wood or oil, and might not immediately associate those costs with
+standard "utilities". If your community is in a flood plain, you might want to include flood
+insurance as an option.
+
+## ALExpense
+
+`ALExpense` is a class that represents an amount that re-occurs periodically, like a monthly
+utility bill, or weekly grocery expenses. It has a few attributes:
+
+* `expense.value` is the actual amount that is reoccurring. For example, someone's rental insurance
+  might always be $50 each month.
+* `expense.times_per_year` is the number of times each year that this amount occurs. If someone has to pay their landlord twice a month, `times_per_year` would be 24. This value can also be fractional if need be; an expense that only needs to be paid once every two years would have a `times_per_year` of 0.5.
+* `expense.source` is the source, or name, of the expense. Food expenses could have a source of `"food"`.
+  * `expense.display_name` is the name of the source that you want to display to the user. This helps with multi-lingual interviews, where you can show the user the name of the expense, but your interview logic can still refer to a single canonical language name. It also helps add or change additional context to the name that you don't want to write out in your interview logic. For example, `expense.source` could be `"medical insurance"`, but `expense.display_name` could be `"Medical Insurance (including health, dental, and vision)"`.
+
+You won't be using `ALExpense` by itself often though; you'll need a list of expenses from the user, which we'll discuss next.
+
+
+## ALExpenseList Tutorial
+
+Let's make a short interview that just asks for someone's expenses. These some pieces
+of code can be used in longer interviews as well.
+
+### Before
+
+Before you start, we'll assume that you:
+
+* have access to a [developer account on a docassemble server](https://suffolklitlab.org/legal-tech-class/docs/classes/assembly-line/2020-assembly-line-assignment-1#before-you-get-started)
+* have the [`ALToolbox` package installed on your server](https://suffolklitlab.org/docassemble-AssemblyLine-documentation/docs/installation#run-the-installation-script)
```

Comment:
```suggestion
* have the [`ALToolbox` or `AssemblyLine` package installed on your server](https://suffolklitlab.org/docassemble-AssemblyLine-documentation/docs/installation#run-the-installation-script)
```
Many people will probably just be using AssemblyLine.

Reply:
Then they'll have `ALToolbox` already installed? I think I understand what you're pointing out; ppl might not immediately recognize the toolbox package, but they would know if they've installed the general assembly line. I might rephrase this differently to address that.

```suggestion
* have the [`AssemblyLine` package installed on your server](https://suffolklitlab.org/docassemble-AssemblyLine-documentation/docs/installation#run-the-installation-script), or have installed the [`ALToolbox` package](https://github.com/SuffolkLITLab/docassemble-ALToolbox) separately.
```